client/src/components/users/UserInformation.vue

```diff
@@ -23,6 +24,19 @@
       @on-enter-keyup="$emit('onEnterKeyup', email)"
       :validator="emailValidator"
     />
+    <div
+      v-else
+      class="disabled-email-address"
+    >
+      <ion-text class="form-label">
+        {{ $msTranslate('CreateOrganization.email') }}
+      </ion-text>
+      <div class="disabled-email-address-value">
```

Comment:
Is there a way to make this take the whole width instead of just half (when greeting the user)?